Mathlib/Data/Matrix/PEquiv.lean

```diff
@@ -156,4 +156,36 @@ theorem equiv_toPEquiv_toMatrix [DecidableEq n] [Zero α] [One α] (σ : Equiv n
     σ.toPEquiv.toMatrix i j = (1 : Matrix n n α) (σ i) j :=
   if_congr Option.some_inj rfl rfl
 
+@[simp]
+lemma map_toMatrix [DecidableEq n] [NonAssocSemiring α] [NonAssocSemiring β]
+    (f : α →+* β) (σ : m ≃. n) : σ.toMatrix.map f = σ.toMatrix := by
+  ext i j
+  simp
+
 end PEquiv
+
+namespace Equiv
+
+open Matrix
+
+variable {α m n : Type*} [DecidableEq m] [DecidableEq n]
+
+lemma toPEquiv_toMatrix_mulVec_single [Fintype n] [NonAssocSemiring α] (σ : m ≃ n) (i : n) (x : α) :
```

Comment:
Strictly speaking this name is backwards; I've created #21108 to fix the existing names that this copies.